tools/analysis_tools/confusion_matrix.py

```diff
@@ -89,14 +98,16 @@ def plot_confusion_matrix(confusion_matrix,
 
     num_classes = len(labels)
     fig, ax = plt.subplots(
-        figsize=(2 * num_classes, 2 * num_classes * 0.8), dpi=180)
+        figsize=(2 * num_classes, 2 * num_classes * 0.8), dpi=300)
```

Comment:
Perhaps we should add a parameter to control the `dpi`, it's too large for ADE20K dataset.